app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -28,6 +30,24 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
+            ((Button) findViewById(R.id.buton)).setText(text);
+
+            if (text == "something") {
+                ((Button) findViewById(R.id.buton)).setText("something");
+            } else           if (text == "something-else") {
+                ((Button) findViewById(R.id.buton)).setText("something-else");
+            }
+        } catch (Exception e) {
+        }
+        findViewById(R.id.buton).setOnClickListener(new OnClickListener() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/c0ca0570c1175a60455e68081fbe6ddf3053938f/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L45"></span>The class com.mathroule.demoapp.MainActivity$2 could be refactored into a named <em>static</em> inner class</td>
    </tr>
  </tbody>
</table>